src/main/scala/part7bigdata/TaxiApplication.scala

```diff
@@ -11,8 +11,6 @@ object TaxiApplication extends App {
     .getOrCreate()
   import spark.implicits._
 
-  val bigTaxiDF = spark.read.load("path/to/your/dataset/NYC_taxi_2009-2016.parquet")
-
```

Comment:
Removed this line of code, because no usage was found. Also, due to this exception was getting raised like this :
Exception in thread "main" org.apache.spark.sql.AnalysisException: [PATH_NOT_FOUND] Path does not exist: file:/M:/rockTheJVM/spark-essentials_main/spark-essentials/path/to/your/dataset/NYC_taxi_2009-2016.parquet. 

After removing this, the application ran successfully.

Reply:
please keep it - the intention was for the student to change the path to wherever they downloaded the parquet files, they can cut it out as necessary